docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -269,7 +273,26 @@ For example, the following command runs tests from the specified file and forces
 testcafe ie tests/sample-fixture.js -e
 ```
 
-### -t \<name\>,  --test \<name\>
+### -c \<factor\>, --concurrency \<factor\>
+
+Runs tests concurrently. The `factor` parameter specifies the number of browser instances
```

Comment:
`Runs tests concurrently` what does it mean? Run all tests three times? Why?

Why do I need this? We must first say that this saves time of tests passing.